schemas/bedrock-authn-token-http.js

```diff
@@ -0,0 +1,171 @@
+/*!
+ * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const postToken = {
+  title: 'postToken',
+  type: 'object',
+  additionalProperties: false,
+  anyOf: [
+    {required: ['email']},
+    {required: ['account']}
+  ],
```

Comment:
Aren't there other properties that are required as well? Looking at the route handler at a quick glance it seems it expects `authenticationMethod`, for example. If you make all of these required, what happens? It seems maybe only `hash` should not be required? We should require clients to set as many of these params as possible to ensure that they are doing what they want; leaving out only those that would create some kind of conflict.

Reply:
We also want to try our best to fail closed here because it's an authn / security API -- we don't want someone to accidentally leave out `requiredAuthenticationMethods` when they really wanted to set it, for example.

Reply:
`authenticationMethod` and `requiredAuthenticationMethods` are not used in any of the tests when calling `post.login`, are all of those incorrect? It does not seem like it's required as it works fine without it.

Reply:
The tests are incorrect.

Reply:
Addressing the tests now, I pushed the update to test on the main branch of another repo.